[2024-05-09T02:06:15.498+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T02:06:15.532+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_v1.training_model_C manual__2024-05-09T02:06:11.499284+00:00 [queued]>
[2024-05-09T02:06:15.543+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_v1.training_model_C manual__2024-05-09T02:06:11.499284+00:00 [queued]>
[2024-05-09T02:06:15.544+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-09T02:06:15.569+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): training_model_C> on 2024-05-09 02:06:11.499284+00:00
[2024-05-09T02:06:15.579+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3379) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-09T02:06:15.582+0000] {standard_task_runner.py:63} INFO - Started process 3385 to run task
[2024-05-09T02:06:15.586+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag_v1', 'training_model_C', 'manual__2024-05-09T02:06:11.499284+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpz88kyqi3']
[2024-05-09T02:06:15.592+0000] {standard_task_runner.py:91} INFO - Job 34: Subtask training_model_C
[2024-05-09T02:06:15.724+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag_v1.training_model_C manual__2024-05-09T02:06:11.499284+00:00 [running]> on host 2b3626ba2b72
[2024-05-09T02:06:15.903+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='satyendra' AIRFLOW_CTX_DAG_ID='my_dag_v1' AIRFLOW_CTX_TASK_ID='training_model_C' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T02:06:11.499284+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T02:06:11.499284+00:00'
[2024-05-09T02:06:15.906+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T02:06:15.907+0000] {python.py:237} INFO - Done. Returned value was: 6
[2024-05-09T02:06:15.908+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T02:06:15.943+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=my_dag_v1, task_id=training_model_C, execution_date=20240509T020611, start_date=20240509T020615, end_date=20240509T020615
[2024-05-09T02:06:16.014+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T02:06:16.073+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T02:06:16.077+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
